### PR TITLE
Port Bay's survival boxes

### PR DESCRIPTION
**Pull request by Arokha**
Includes some tweaks to items that usually come in the boxes to make them more similar to the Bay implementation. Also includes new RPED sprite For Free:tm: (the old one is pretty scary okay)

image
(Picture old, only one survival box icon is used (the others are in there if you want them))

image

Tweaks:
* Air tanks are generated based on the species breath_type, so there's not quite as much variety in box definitions
* Adds the `start_with` list to storage objects, allowing you to specify what objects they start with easily without overriding their New() (I'll make a separate PR where I convert the existing ones)
* Pilots, SAR, Explorers get the 'comprehensive' ones above. Everyone else gets an old one.

### CODE REVIEW

File: code/game/objects/items/weapons/storage/boxes.dm [hunk 1]
```diff
@@ -52,20 +52,44 @@
...
 	new /obj/item/weapon/tank/emergency/oxygen(src)
-
+	new /obj/item/weapon/reagent_containers/hypospray/autoinjector(src)
+	new /obj/item/stack/medical/bruise_pack(src)
+	new /obj/item/device/flashlight/glowstick(src)
+	new /obj/item/weapon/reagent_containers/food/snacks/candy/proteinbar(src)
+
+/obj/item/weapon/storage/box/vox
+	name = "vox survival kit"
+	desc = "A box decorated in warning colors that contains a limited supply of survival tools. The panel and black stripe indicate this one contains nitrogen."
```
Comment (Cyantime) [inline code/game/objects/items/weapons/storage/boxes.dm]: Should be plasma, not nitrogen. Unless you want all the vox to suffocate, which I support.

Comment (Arokha) [reply code/game/objects/items/weapons/storage/boxes.dm]: Fixed.

**Comment by Anewbe** [comment]
Definitely thought I commented on this!

Any chance I can get them as additions to the selection, rather than replacements?

**Comment by Arokha** [comment]
Added to what selection? There isn't one for survival boxes.

**Comment by Anewbe** [comment]
Wording error on my part.

I don't want to replace every survival box with these, at least just yet. I was thinking of stealing/having them specifically for Explorers, and possibly off-station antags?

**Comment by Arokha** [comment]
Ah. Survival boxes are currently done by a proc on the species. I dunno. It could take another var that's like 'comprehensive box' or something I guess.

**Comment by Anewbe** [comment]
The outfit code might already have a system to check that? I'm pretty sure it works for Engineers, at least?

**Comment by Arokha** [comment]
I altered this so that it's more robust than before. Updated the PR desc to explain.

**Pull request merged**